builder/build_steps/build_app.yaml.in

```diff
@@ -1,6 +1,6 @@
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '--pull', '--no-cache', '-t', '$IMAGE', '-f', 'build_app/Dockerfile', '.']
+    args: ['build', '--pull', '--no-cache', '-t', 'gcr.io/$PROJECT/ruby-build-app:$TAG', '-f', 'build_app/Dockerfile', '.']
```

Comment:
The retagging process copies images from gcr.io to all the localized domains. This build script assumes runtime build steps will undergo the same retagging process that the base image currently does.

You make a good point, though. We should verify with @dlorenc that we will retain the same retagging process given the switch to versioned builders. I think having the script conform to the current process (as it does here) will help drive that decision.